src/commands/init.js

```diff
@@ -7,33 +7,32 @@ const path = require('path');
 
 class InitCommand extends ApifyCommand {
     async run() {
-        if (getLocalConfig()) {
-            throw new Error('File apify.json already exists in current directory.');
-        }
-
         const { args } = this.parse(InitCommand);
-        let { actName } = args;
+        let { actorName } = args;
         const cwd = process.cwd();
 
-        if (!actName) {
-            const answer = await inquirer.prompt([{ name: 'actName', message: 'Actor name:', default: path.basename(cwd) }]);
-            ({ actName } = answer);
+        if (getLocalConfig()) {
+            outputs.warning('Skips creating of apify.json, file already exists in the directory.');
```

Comment:
`Skipping creation of apify.json, the file already exists in the current directory.`